src/processor/processor.module.ts

```diff
@@ -46,6 +47,17 @@ import { NonceService } from './nonce.service';
     }),
     BullModule.registerQueue({
       name: 'graphUpdateQueue',
+      defaultJobOptions: {
+        attempts: 3,
+        backoff: {
+          type: 'exponential',
+        },
+        removeOnComplete: false,
```

Comment:
This was `true` before... are we sure we don't want to keep it true? The Redis instance will get quite large with a record of all the completed jobs...